common-pipeline/datarecord-metadata/datarecord-metadata-v3-schema.json

```diff
@@ -166,9 +169,40 @@
         "runtime_type": {
           "description": "Type of runtime",
           "type": "string"
+        },
+        "fraction_digits": {
+          "description": "Microsecond length for time and timestamp",
+          "type": "number"
+        },
+        "is_key": {
+          "description": "A field is key",
+          "type": "boolean"
+        },
+        "is_signed": {
+          "description": "Signed or Unsigned for number (bigint, integer, smallint, tinyint)",
```

Comment:
I'm not sure we should include `(bigint, integer, smallint, tinyint)`.  Those would be specific to the application.

Reply:
Fixed.